x-pack/legacy/plugins/uptime/public/components/functional/snapshot_heading.tsx

```diff
@@ -13,27 +13,17 @@ interface Props {
   total: number;
 }
 
-const getMessage = (down: number, total: number): string => {
-  if (down === 0 && total > 0) {
-    return i18n.translate('xpack.uptime.snapshot.zeroDownMessage', {
-      defaultMessage: 'All monitors are up',
-    });
-  } else if (down === 0 && total === 0) {
-    return i18n.translate('xpack.uptime.snapshot.noMonitorMessage', {
-      defaultMessage: 'No monitors found',
-    });
-  }
-  return i18n.translate('xpack.uptime.snapshot.downCountsMessage', {
-    defaultMessage: '{down}/{total} monitors are down',
-    values: {
-      down,
-      total,
-    },
-  });
-};
+export const SnapshotHeading = ({ total }: Props) => {
+  const monitorsText =
```

Comment:
This code will display `0 Monitors` when there's nothing defined. I just wanted to point this out. Do we know if there is any concern with copy showing info like this vs. `No monitors`? I think it's probably ok but we might want to check with a tech writer.

Reply:
I don't think this is an issue, it's proper English.